velox/functions/prestosql/aggregates/tests/MinMaxByAggregationTest.cpp

```diff
@@ -1127,6 +1127,23 @@ TEST_F(MinMaxByComplexTypes, map) {
       {data}, {}, {"min_by(c0, c1)", "max_by(c0, c1)"}, {expected});
 }
 
+TEST_F(MinMaxByComplexTypes, limits) {
+  static const auto kMax = std::numeric_limits<int64_t>::max();
+  static const auto kMin = std::numeric_limits<int64_t>::min();
+  auto data = makeRowVector({
+      makeFlatVector<int64_t>({1, 2}),
```

Comment:
Would it be possible to test this for all supported types?

Reply:
Hit a problem when add Timestamp test https://github.com/facebookincubator/velox/issues/5199